base/docs/bootstrap.jl

```diff
@@ -4,7 +4,15 @@ module DocBootstrap
 
 export @doc
 
-const docs = []
+type List
+    head
+end
+type Node
+    data
+    next
+end
```

Comment:
Sure, sounds fine. I did it like that so that it would be ``const``, but probably doesn't matter for this.